lib/kramdown-rfc2629.rb

```diff
@@ -56,8 +62,8 @@ def parse_xref
       # Introduce new (((target))) syntax for irefs
       def parse_iref
         @src.pos += @src.matched_size
-        href = @src[1]
-        el = Element.new(:iref, nil, {'target' => href}) # XXX
+        anchor, = RFC2629Kramdown.split_xref(@src[1])
+        el = Element.new(:iref, nil, {'target' => anchor}) # XXX
```

Comment:
Yep, you are right.  I was hoping to take advantage of the cleanup that is done in the split function, but it changes the entries in the index in a bad way.